src/Donations/Components/ContactsForm.tsx

```diff
@@ -19,6 +19,7 @@ import { AddressCandidate, GeocodeSuggestion } from "src/Common/Types/arcgis";
 import GiftIcon from "public/assets/icons/GiftIcon";
 import { euCountries } from "src/Utils/countryUtils";
 import { isEmailValid } from "src/Utils/isEmailValid";
+// import { DevTool } from "@hookform/devtools";
```

Comment:
Maybe just remove this line or do you want to keep it as some form of documentation on how to debug?